extensions/kogito/deployment/src/main/java/io/quarkus/kogito/deployment/KogitoAssetsProcessor.java

```diff
@@ -74,12 +73,12 @@ public void generateModel(ArchiveRootBuildItem root,
 
     private boolean hotReload(LaunchMode launchMode) {
         if (launchMode == LaunchMode.DEVELOPMENT) {
-            Path kiePath = Paths.get("devmode.kie");
```

Comment:
@mariofusco what is this devmode.kie (or now devmode.kogito) used for?

Reply:
@mswiderski this is just an empty marker file to avoid regenerating the whole model during hot reload (since the processor is invoked again by quarkus). You cannot add a field in the processor (quarkus will throw an exception if you try to do so), but I think I could replace it with a static field and actually I will do so.